Umee-Bots/Lending-Pool-Reentrancy/src/agent.config.ts

```diff
@@ -0,0 +1,24 @@
+import { AgentConfig } from "./utils";
+
+const CONFIG: AgentConfig = {
+  // Signatures that will result in a finding emission if called in a reentrant manner
+  reentrancyBlacklist: [
+    "borrow(address,uint256,uint256,uint16,address)",
+    "withdraw(address,uint256,address)",
+    "flashLoan(address,address[],uint256[],uint256[],address,bytes,uint16)",
+    "deposit(address,uint256,address,uint16)",
+    "repay(address,uint256,uint256,address)",
+    "swapBorrowRateMode(address,uint256)",
+    "rebalanceStableBorrowRate(address,address)",
+    "setUserUseReserveAsCollateral(address, bool)",
+    "liquidationCall(address,address,address,uint256,bool)",
+    "finalizeTransfer(address,address,address,uint256,uint256,uint256)",
+  ],
+  // Address of the LendingPool contract
+  lendingPoolAddress: "0xce744a9baf573167b2cf138114ba32ed7de274fa",
+};
+
+// Uncomment these lines for the testnet test:
```

Comment:
just grammatical concern: 
// Uncomment `this line` for the testnet test:
if there's only 1 line. otherwise it might feel like something is missing.